### PR TITLE
6713 only show one flash message at a time on transaction page

### PR DESCRIPTION
**Pull request by X0nic**

### CODE REVIEW

**Comment by hooverlunch** [comment]
I think we can perhaps ignore the failing spec here for now until everything is integrated. There are just so many moving pieces it's not clear what's causing this failure.

**Comment by emitche** [comment]
@X0nic: Please merge develop into this branch and resolve any merge conflicts. Thank you!

**Comment by X0nic** [comment]
This should match what you need @linesarefuzzy . Ping me if something jumps out at you. I had real problems trying to test this with live QBO data.

File: app/models/division.rb [hunk 1]
```diff
@@ -138,6 +138,7 @@ def accounts
   end
 
   def qb_accounts_connected?
+    return true if Rails.env.test?
```
Comment (X0nic) [inline app/models/division.rb]: If anyone has a better way of mocking this for flow specs?

**Comment by hooverlunch** [comment]
This should be good to go if the specs pass. I'll be watching.

@linesarefuzzy There will probably be some ugly merge conflicts with the integration work and this work (once merged to develop).

**Comment by emitche** [comment]
The add transaction button should not show up when QuickBooks is connected.

image

**Comment by emitche** [comment]
When transactions do not exist and QuickBooks is connected, should we be seeing this? Or should there be a message that says there are no transactions?

image

**Comment by emitche** [comment]
Redmine#7489 was created to address these questions.

**Pull request merged**